vowpalwabbit/parse_args.cc

```diff
@@ -1003,6 +1006,13 @@ void parse_reductions(vw& all)
   all.reduction_stack.push_back(bs_setup);
 
   all.l = setup_base(all);
+
+  if (all.probabilities)
+  { if (!all.vm.count("oaa") && !all.vm.count("csoaa_ldf"))
```

Comment:
Simply: someone uses `--probabilities` without `--oaa` or `--csoaa_ldf`. The only other reduction I am aware of, where `--probabilities` could be used is binary classification, but here we already can use `--link=logistic`, which has the same effect of reporting probabilities. Nevertheless, so far `--probabilities` are implemented only for oaa and csoaa_ldf, so I think it is better warn users if they try to use it elsewhere.